lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -432,9 +432,16 @@ bool pullCustomize(const PullImportPaths& importedPaths, const UsdMayaPrimUpdate
         const auto&       pulledUfePath = *ufePathIt;
         MFnDependencyNode dgNodeFn(dagPath.node());
 
+        // If the Maya node holds USD type information (e.g. a dummy transform
```

Comment:
I think a "no" is missing in the sentence: "If the Maya node holds *no* USD type informtation..."?

Reply:
No, the sentence is correct: we look at the Maya node, and if there is USD type information recorded in it, we will use the USD type to do updater lookup, rather than use the Maya type.  If you have a wording to suggest that would make this clearer, let me know!

Reply:
Ah OK. Maybe "instead of the Maya type", because I had read "instead" as "instead of the USD type".